wizard-js/README.md

```diff
@@ -0,0 +1,24 @@
+# Node.js AST snippet labeler
+
+This script parses Node.js snippets, and takes advantage of DPE idioms to automatically map tests to their respective snippets.
+
```

Comment:
Can you add more info why this is being used for? Know limitations?

Reply:
Updated the README - PTAL?